understanding/no-keyboard-trap.html

```diff
@@ -1,18 +1,16 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="ja" xml:lang="ja">
    <head>
       <meta charset="UTF-8" />
       <meta name="viewport" content="width=device-width, initial-scale=1" />
-      <title>Understanding Success Criterion  2.1.2: No Keyboard Trap | WAI | W3C</title>
+      <title>解説書 達成基準  2.1.2: キーボードトラップなし | WAI | W3C</title>
```

Comment:
ものすごく細かいですが、せっかくなのでスペースを1つにしてしまってよいかなと。
```suggestion
      <title>解説書 達成基準 2.1.2: キーボードトラップなし | WAI | W3C</title>
```